.jshintrc

```diff
@@ -9,6 +9,7 @@
   "quotmark": "single",
   "undef": true,
   "unused": true,
+  "esversion": 6,
```

Comment:
Why this addition?

Reply:
In the roadmap of the IoT Agents was described to start the process to migrate to ES6 (ECMAScript 2015). I started it in the lwm2mAgent.js and I added this line to allow IDEs to understand the new changes.

Reply:
Roadmap tells: 

> Incremental introduccion of ECMAScript6 syntax (previous analysis of which sub-set of interesting aspect we want to take)

We are not yet conducted such analysis... however, I understand the changes regarding ECMAScript6 in this PR are small ones so it's fine.

NTC